tests/context/conftest.py

```diff
@@ -242,3 +242,900 @@ def ce_details_dynamic_event_deleted(ce_details_dynamic_event):
+                {
+                    "sampleLinkPK": 3,
+                    "collectionExerciseId": "f44feba3-c6e8-4e5f-a7ce-065ddc081424",
+                    "sampleSummaryId": "0b35dff1-07a3-4cfc-9eaf-2f01f6602e11",
+                }
+            ],
+            "responseStatus": "In progress",
+            "companyName": "RUNAME1_COMPANY1 RUNNAME2_COMPANY1 ",
+            "companyRegion": "GB",
+            "tradingAs": "TOTAL UK ACTIVITY  ",
+        },
+    ]
+
+
+@pytest.fixture()
+def reporting_unit():
+    return {
+        "associations": [
+            {
+                "businessRespondentStatus": "ACTIVE",
+                "enrolments": [{"enrolmentStatus": "ENABLED", "surveyId": "02b9c366-7397-42f7-942a-76dc5876d86d"}],
+                "partyId": "bf19a18f-fe15-4005-b698-fdd36f35f940",
+            }
+        ],
+        "attributes": {
+            "birthdate": "01/09/1993",
+            "cellNo": 7,
+            "checkletter": "F",
+            "currency": "S",
+            "entname1": "ENTNAME1_COMPANY1",
+            "entname2": "ENTNAME2_COMPANY1",
+            "entname3": "",
+            "entref": "9900000576",
+            "entrepmkr": "E",
+            "formType": "0001",
+            "froempment": 8478,
+            "frosic2007": "45320",
+            "frosic92": "50300",
+            "frotover": 801325,
+            "inclexcl": "D",
+            "legalstatus": "1",
+            "name": "RUNAME1_COMPANY1 RUNNAME2_COMPANY1",
+            "region": "FE",
+            "runame1": "RUNAME1_COMPANY1",
+            "runame2": "RUNNAME2_COMPANY1",
+            "runame3": "",
+            "rusic2007": "45320",
+            "rusic92": "50300",
+            "sampleUnitId": "d8579750-8817-4a58-8fa5-867f1447e1cc",
+            "seltype": "C",
+            "trading_as": "TOTAL UK ACTIVITY",
+            "tradstyle1": "TOTAL UK ACTIVITY",
+            "tradstyle2": "",
+            "tradstyle3": "",
+        },
+        "id": "a5348157-feb4-4bad-9614-fc76e2bfea94",
+        "name": "RUNAME1_COMPANY1 RUNNAME2_COMPANY1",
+        "sampleSummaryId": "d646f19b-827f-4934-a71a-43ba458045b6",
+        "sampleUnitRef": "49900000001",
+        "sampleUnitType": "B",
+        "trading_as": "TOTAL UK ACTIVITY",
+    }
+
+
+@pytest.fixture
+def multiple_reporting_units():
+    return {
+        "associations": [
+            {
+                "businessRespondentStatus": "ACTIVE",
+                "enrolments": [{"enrolmentStatus": "ENABLED", "surveyId": "02b9c366-7397-42f7-942a-76dc5876d86d"}],
+                "partyId": "258b95c5-1e48-4357-881c-ad4d544eab32",
+            },
+            {
+                "businessRespondentStatus": "ACTIVE",
+                "enrolments": [{"enrolmentStatus": "ENABLED", "surveyId": "02b9c366-7397-42f7-942a-76dc5876d86d"}],
+                "partyId": "985bf97e-4f03-4898-92ef-dd7aac23ab08",
+            },
+        ],
+        "attributes": {
+            "birthdate": "01/09/1993",
```

Comment:
This again feels like it can be extracted and used as a fixture import

Reply:
Made into a fixture import